v1/format/format_test.go

```diff
@@ -907,6 +907,34 @@ p contains x if {
 	}
 }
 
+func TestGroupableOneLinerRules(t *testing.T) {
+	contents := []byte(`package test
+
+foo := 1 if input.x
+foo := 2 if not input.x
+
+a := 1
+b := 2
+
+c := 3
+
+d := 4
+
+# comment above group
+e := 5
+f := 6
```

Comment:
we respect partial set groupings too right? E.g.
```suggestion
e := 5
f := 6
g contains 7

h contains 8

i contains 9
j contains 10

k contains 11
```